frontend/src/views/HomeView.vue

```diff
@@ -2,11 +2,14 @@
 import { useRouter } from 'vue-router';
 import { Button } from '@/lib/primevue';
 import { RouteName } from '@/utils/enums/application';
+import { useAuthZStore } from '@/store';
+import { NavigationPermission } from '@/store/authzStore';
 
 // Store
 const router = useRouter();
 const toHousing = (): void => {
-  router.push({ name: RouteName.HOUSING });
+  if (useAuthZStore().canNavigate(NavigationPermission.INT_HOUSING)) router.push({ name: RouteName.INT_HOUSING });
+  else router.push({ name: RouteName.EXT_HOUSING });
```

Comment:
users will have either ext and int navigation permissions but on the off chance they have neither should we be checking that they have ext permissions and not just an else. If neither do nothing or show toast showing say you have no permissions or something